Makefile

```diff
@@ -60,6 +60,8 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	dpkg-source -x $(DSC_FILE)
 
 	pushd $(BUILD_DIR)
+	sed -i '/abiname:.*/ s/$$/-1/' debian/config/defines
+	fakeroot debian/rules debian/control-real || true
```

Comment:
why make it always true here?

Reply:
When "fakeroot" command is run it displays a message saying "the message is intentional please ignore and continue" so true is required to continue.

Reply:
When "fakeroot" command is run it displays a message saying "the message is intentional please ignore and continue" so true is required to continue.